doc/concepts/concept_embeddings.rst

```diff
@@ -0,0 +1,65 @@
+
+.. math:: | \mathcal{D} \rangle = \frac{1}{\sqrt{2}}|01 \rangle + \frac{1}{\sqrt{2}} |11 \rangle.
+
+.. note:: For :math:`N` bits, there are :math:`2^N` possible basis states. Given :math:`M \ll 2^N`, the basis embedding of :math:`\mathcal{D}` will be sparse :cite:`schuld2018supervised`. 
+
+
+Amplitude Embedding
+^^^^^^^^^^^^^^^^^^^^
+
+In the amplitude-embedding technique, data is encoded into the amplitudes of a quantum state. A normalized classical :math:`N`-dimensional datapoint :math:`x` is represented by the amplitudes of a :math:`n`-qubit quantum state :math:`| \psi_x \rangle` as
+
+.. math:: | \psi_x \rangle = \sum_{i=1}^{N} x_i |i \rangle,
+
+where :math:`N=2^n`, :math:`x_i` is the :math:`i`-th element of :math:`x` and :math:`| i \rangle` is the :math:`i`-th computational basis state. For example, let's say we want to encode :math:`x=1010` using amplitude embedding. The first step is to normalize it, i.e., :math:`x_{norm}=\frac{1}{\sqrt{2}}(1010)`. The corresponding amplitude encoding uses two qubits to represent :math:`x_{norm}` as
+
+.. math:: | \psi_{x_{norm}} \rangle = \frac{1}{\sqrt{2}}|00 \rangle + \frac{1}{\sqrt{2}}|10 \rangle.  
+
+Let's consider the classical dataset :math:`\mathcal{D}` mentioned above. Its amplitude embedding can be easily understood if we concatenate all the input examples :math:`x^{(m)}` together into one vector, i.e., 
+
+.. math:: \alpha = C_{norm} \{ x^{(1)}_1, \ldots, x^{(1)}_N, x^{(2)}_1, \ldots, x^{(2)}_N, \ldots, x^{(M)}_1, \ldots, x^{(M)}_N \},
+ 
+where :math:`C_{norm}` is the normalization constant; this vector must be normalized :math:`|\alpha|^2=1`. The input dataset can now be represented in the computational basis as
+
+.. math:: | \mathcal{D} \rangle = \sum_{i=1}^{2^n} \alpha_i |i \rangle,
+
+where :math:`\alpha_i` are the elements of amplitude vector :math:`\alpha` and :math:`| i \rangle` are the computational basis states. The number of amplitudes to be encoded are :math:`N \times M`. As a system of :math:`n` qubits provides :math:`2^n` amplitudes, **amplitude embedding requires** :math:`\bm{n \geq \log_2({NM})}`  **qubits.**
+
+
+.. note:: If the total number of amplitudes to embed, i.e., :math:`N \times M`, is less less than :math:`2^n`, *non-informative* constants can be *padded* to :math:`\alpha `:cite:`schuld2018supervised`. For example, if we have three examples with two features each, we have six amplitudes to embed. However, as we have to use at least three qubits, we have to concatenate two constants at the end of :math:`\alpha`. 
+
+.. important:: There are many other embedding techniques with similar embedding protocols. For example, *Squeezing embedding* and *Displacement embedding* are used with continuous-variable quantum computing models, where classical information is encoded in the squeezing and displacement operator parameters. *Hamiltonian embedding* uses an implicit technique by encoding information in the evolution of a quantum system :cite:`schuld2018supervised`.  
```

Comment:
```suggestion
.. important::

    There are many other embedding techniques with similar embedding protocols.
    For example, *squeezing embedding* and *displacement embedding* are used
    with continuous-variable quantum computing models, where classical information
    is encoded in the squeezing and displacement operator parameters.
    *Hamiltonian embedding* uses an implicit technique by encoding information in the
    evolution of a quantum system :cite:`schuld2018supervised`.  
```
You could also make 'squeezing embedding' and 'displacement embedding' links to the functions in the embeddings module, or add them to the see also below. But I'm on the fence, might be best to keep links to the code documentation minimal from the concepts section.